### DISCUSSION TITLE
Cat allergies

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#379] [General]
What treatment is the best from your experience?
Should I just wait or is it better to start with the meds immediately?

**Comment by hernessteve** [comment]
First step here - call your vet and discuss the options.
You might wanna also read an article about benadryl and cats [https://www.catfoodpoint.com/benadryl-for-cats/](url).
It all depends on the symptoms your cat having and how is her/his condition.
